include/eredis.hrl

```diff
@@ -1,6 +1,7 @@
 %% Public types
 
 -type reconnect_sleep() :: no_reconnect | integer().
+-type process_name() :: {local, atom()} | {global, term()} | {via, atom(), term()}.
```

Comment:
Maybe `process_name()` is not the best name for this type, I'm open to suggestions.

Reply:
I suggest `registered_name()`.